externals/imguizmo/ImGuizmo.hpp

```diff
@@ -0,0 +1,166 @@
+// https://github.com/CedricGuillemet/ImGuizmo
```

Comment:
This and ImGuizmo.cpp should not be included in the project; they are part of an external dependency we can track via git submodules.